File: credit_check.py
def credit_check(number):
  reversed_list = str(number)[::-1]
  multiplied_list = []
  summed_digits = []
   
  for index, item in enumerate(reversed_list):
    if(index % 2 == 0):
        multiplied_list.append(int(item))
    else:
        multiplied_list.append(int(item) * 2)
  for i in multiplied_list:
    if(i > 9):
        summed_digits.append(i - 9)
    else:
        summed_digits.append(i)
    summed_list = sum(summed_digits) 
  if(summed_list % 10 == 0):
    print(f"{number} is a valid credit card number")
  else:
    print(f"{number} is not a valid credit card number")
    
credit_check(5541808923795240)

File: creditCheck.js
function creditCheck(number){
let reversedArray = number.toString().split('').reverse()
let multipliedArray = []
let summedDigits = []

for(let i = 0; i < reversedArray.length; i++){
  if([i] % 2 === 0){
  multipliedArray.push(parseInt(reversedArray[i]))
  } else {
    multipliedArray.push(reversedArray[i] *= 2)
  }
}
for(let i = 0; i < multipliedArray.length; i++){
    if(multipliedArray[i] > 9){
    summedDigits.push(multipliedArray[i] - 9)
  } else {
    summedDigits.push(multipliedArray[i])
  }
}
let summedArray = summedDigits.reduce((a, b) => a + b, 0)
if(summedArray % 10 === 0){
    console.log(`${number} is a valid credit card number`)
  } else {
    console.log(`${number} is not a valid credit card number`)
  }
}
creditCheck(5541808923795240)
